src/unk_0202631C.c

```diff
@@ -1139,10 +1135,10 @@ void sub_02027454(PokedexData *param0)
     param0->nationalDexObtained = 1;
 }
 
-BOOL sub_02027474(const PokedexData *param0)
+BOOL Pokedex_NationalDexObtained(const PokedexData *pokedex)
```

Comment:
```suggestion
BOOL Pokedex_IsNationalDexObtained(const PokedexData *pokedex)
```

I'd argue more for `Pokedex_IsNationalMode`, but that doesn't totally align with the member field.